src/atc/process/etl_call_args.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+from atc.utils.job_parameter_helper import JobParameterHelper
+
+
+@dataclass
+class ETLCallArgs:
+    from_date: datetime = None
+    to_date: datetime = None
+    stream_start_date: datetime = None
+    load_mode: str = None
+
+    def get_call_args(self):
```

Comment:
So the arguments that you can get from your job are only exactly these 4? It seems very specific to your use-case. Maybe you should extract the contstructor arguments from the EtlParameters object? Maybe this generalization is for a later PR.

Reply:
For now, yes the get_call_args() fetches these four arguments. However, if I want to add more call arguments it will be easy to implement and allow to be used across all inheriting flows.